src/main/java/com/playmonumenta/networkchat/commands/chat/channel/ChatChannelDescriptionCommand.java

```diff
@@ -0,0 +1,45 @@
+package com.playmonumenta.networkchat.commands.chat.channel;
+
+import com.playmonumenta.networkchat.ChannelManager;
+import com.playmonumenta.networkchat.ChannelPredicate;
+import com.playmonumenta.networkchat.NetworkChatProperties;
+import com.playmonumenta.networkchat.commands.ChatCommand;
+import com.playmonumenta.networkchat.utils.CommandUtils;
+import dev.jorel.commandapi.arguments.Argument;
+import dev.jorel.commandapi.arguments.LiteralArgument;
+import dev.jorel.commandapi.exceptions.WrapperCommandSyntaxException;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.command.CommandSender;
+
+public class ChatChannelDescriptionCommand {
+	public static void register() {
+		if (NetworkChatProperties.getChatCommandDescriptionEnabled()) {
+			Argument<String> newChannelDescription = ChannelManager.getChannelDescriptionArgument("New Channel Description");
+			Argument<String> channelName = ChannelManager.getChannelNameArgument("Channel Name", ChannelPredicate.MAY_MANAGE);
+
+			// When changing a description, it should also be changed in
+			// epic/data/datapacks/base/data/monumenta/functions/chat/formatting.mcfunction
+
+			ChatCommand.getBaseCommand()
+				.withArguments(new LiteralArgument("channel"))
+				.withArguments(new LiteralArgument("description"))
+				.withArguments(channelName)
+				.withArguments(newChannelDescription)
+				.executesNative((sender, args) -> {
+					if (!CommandUtils.hasPermission(sender, "networkchat.description")) {
```

Comment:
The general permission structure is `networkchat.<what scope is being updated>.<property being edited>`, like `networkchat.channel.description`.